tests/test_about.py

```diff
@@ -15,17 +15,25 @@
 Unit tests for the :mod:`pennylane` configuration classe :class:`Configuration`.
 """
 # pylint: disable=protected-access
+from io import StringIO
+import re
+from unittest.mock import patch
+
 import pennylane as qml
 import pytest
-import re
 
 
-def test_about(capfd):
+def test_about():
     """
     about: Tests if the about string prints correct.
     """
     qml.about()
-    out, err = capfd.readouterr()
+
+    with patch("sys.stdout", new=StringIO()) as redirect:
+        qml.about()
+
+    out = redirect.getvalue().strip()
+
```

Comment:
What's the story here? 🤔

Reply:
TensorFlow uses the abseil package, which replaces the default Python logging interface as v0.7.

There were lots of complaints about this, because it breaks peoples own configured logging modules. Another side effect I discovered was that it breaks the `capfd` and `caplog` fixtures in pytest, which would cause the `test_about.py` and `test_configuration.py` tests to crash with no traceback.

They just pushed a v0.8 of abseil which attempts to fix this (so now `caplog` works), but as far as I can tell `capfd` is still broken by v0.8 :slightly_frowning_face: